<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shezhi</class>
 <widget class="QWidget" name="shezhi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>791</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>波特率</string>
        </property>
        <widget class="QRadioButton" name="rate1">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>108</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>9600</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rate2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>126</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>57600</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rate3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>152</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>115200</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rate4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>193</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>460800</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="auto_mode">
        <property name="title">
         <string>无人值守模式</string>
        </property>
        <widget class="QRadioButton" name="autodspf">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>40</y>
           <width>37</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>关</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="autodspt">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>37</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>开</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>花盆数目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="devices">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>采样周期(分钟)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dsptimer">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>上报周期(小时)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dtutimer">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="save">
        <property name="text">
         <string>保存(重启生效)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
